components/shared/main-nav.js

```diff
@@ -10,11 +10,15 @@ const styles = {
     root: {
         flexGrow: 1,
         backgroundColor: '#003da5',
+        margin: '0 auto',
+        width: '100%',
     },
     navBar: {
         boxShadow: 'none!important',
-        maxWidth: 1200,
+        Width: '100%',
```

Comment:
lowercase `width`
<img width="1144" alt="Screen Shot 2019-09-05 at 6 07 50 PM" src="https://user-images.githubusercontent.com/19806441/64393824-232d5500-d008-11e9-834d-b8ae8d239ca9.png">
the 100% is not being applied right now